st_package_reviewer/__main__.py

```diff
@@ -40,7 +40,8 @@ def _prepare_nargs(nargs):
 
 
 def main():
-    """\
+    """Main.
```

Comment:
This docstring is later used to generate the help message, which is sub-optimal but works. Your change introduced a random "Main." in the help message.